src/js/components/SchemaForm.js

```diff
@@ -174,36 +177,28 @@ class SchemaForm extends mixin(StoreMixin, InternalStorageMixin) {
   }
 
   getRemoveRowButton(generalDefinition, prop, id, title = null) {
+    let deleteButton = (
+      <div key={`${prop}${id}-remove`} className="align-self-flex-end">
```

Comment:
the reason i didn't do it this way was because this `div` needs the class `form-row-element` if it's by itself and doesn't need it if it's within the title